src/Cherry/Stage/Parse/Expression/Identifier.elm

```diff
@@ -1,6 +1,6 @@
 module Cherry.Stage.Parse.Expression.Identifier exposing 
     ( parser
-    , nameParser, namespaceParser
+    , nameParser, namespaceParser, objectFieldParser
```

Comment:
This doesn't need to be exposed, because the main `parser` already uses this one when appropriate. Try and keep things exposed unless absolutely necessary.